docs/reference/api-overview.md

```diff
@@ -66,26 +73,27 @@ Currently, there are several API groups in use:
 
 *  The *core* (also called *legacy*) group, which is at REST path `/api/v1` and is not specified as part of the `apiVersion` field, for example, `apiVersion: v1`.
 *  The named groups are at REST path `/apis/$GROUP_NAME/$VERSION`, and use `apiVersion: $GROUP_NAME/$VERSION`
-   (for example, `apiVersion: batch/v1`).  Full list of supported API groups can be seen in [Kubernetes API reference](/docs/reference/).
+   (for example, `apiVersion: batch/v1`). You can find the full list of supported API groups in [Kubernetes API reference](/docs/reference/).
 
-There are two supported paths to extending the API with [custom resources](/docs/concepts/api-extension/custom-resources/):
+The two paths that support to extend the API with [custom resources](/docs/concepts/api-extension/custom-resources/) are:
 
 1. [CustomResourceDefinition](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/)
-   is for users with very basic CRUD needs.
-1. Coming soon: users needing the full set of Kubernetes API semantics can implement their own apiserver
-   and use the [aggregator](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md)
-   to make it seamless for clients.
+   is for basic CRUD needs.
+1. Coming soon: for the full set of Kubernetes API semantics to implement their own apiserver
```

Comment:
I think you can drop the "Coming soon:" - since this was originally written (or last updated), API Aggregation has definitely been established as a path forward. Having recently done some client programming, I'm not sure that I'd claim it as "seamless" in hindsight, but I won't quibble on the aspirations of the effort.